samples/boards/stm32/power_mgmt/adc/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
```

Comment:
Did you check behavior with `CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=y`

Reply:
I didn't see any changes in power consumption with exclusive enabled